fullstack-cert/typescript-projects/generics-workshop/src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Generics Lab</title>
```

Comment:
This should probably be changed to better reflect the project like `Bug Form` or something like that.

Reply:
Renamed to `Bug Emoji Picker`.